docs/development_suite/api-console/api-design/flow-manager-configurator/how-it-works.md

```diff
@@ -15,76 +20,85 @@ In order to describe the functionalities of the tool let's split the interface i
 
-The history buttons are placed on the right side of the header. They provide the standard history actions: *undo* and *redo*. The history is based on actions that result in configuration changes: the maximum supported length is 10 actions.
+- **History Buttons**: the history buttons are placed on the right side of the header. They provide the standard history actions: *undo* and *redo*. The history is based on actions that result in configuration changes: the maximum supported length is 10 actions.
 
-### Settings
+- **Settings**
 
 The settings button is placed on the right side of the header. It provides a dropdown menu with the following options:
 - Persistency Management
-- Deep Merge (only for Flow Manager version >= `2.6.0`)
+- Service configuration
+- Service settings
 - Unlink microservice (only for linked services)
+- New layout (only if you has not switch to the new layout yet)
 
 #### Persistency Management
 
 The persistency management modal allows to configure the persistency method used by the Flow Manager service. The displayed types depend on the Flow Manager version as described [here](/runtime_suite/flow-manager-service/changelog.md).
 
 ![Persistency Management](img/persistency-management.png)
 
-#### Deep Merge
+#### Service configuration
 
-The deep merge modal allows to configure the deep merge setting of the Flow Manager as described [here](/runtime_suite/flow-manager-service/30_configuration.md#settings).
+The Service Configuration modal allows you to download and copy the configuration of the Flow Manager Service, regardless of whether the service is linked to the configurator.
+
+#### Service settings
+
+The service settings modal includes:
+
+- Deep merge: allows to configure the deep merge setting of the Flow Manager as described [here](/runtime_suite/flow-manager-service/30_configuration.md#settings).
 
 :::info
 This feature is available starting from Flow Manager version `2.6.0`.
 :::
 
-![Deep Merge](img/deep-merge.png)
+- Events: Enabling the Events toggle configures the Flow Manager to respond with an error if an unexpected event is received. If the event is as expected, the Flow Manager will respond with an "OK" status.
+
+:::info
+This feature is available starting from Flow Manager version `2.6.5`.
+:::
 
 #### Unlink Microservice
 
 The unlink microservice modal alerts the user before actually performing the [unlink process](/development_suite/api-console/api-design/flow-manager-configurator/overview.md#unlink).
 
 ![Unlink Service](img/unlink-service.png)
 
+#### New Layout
+
+The New Layout modal allows you to switch to the new layout, which includes the canvas view for external executors. Once you switch to the new layout, this action is irreversible, and the option to switch will no longer be visible.
```

Comment:
Forse è meglio che la parte dove diciamo che l'azione è irreversibile la mettiamo in un warning(o danger) cosi a livello visivo si vede di più e magari non si rischia che l'utente se la perdi questa informazione